File: C++/prime_number.cpp
#include <iostream>
using namespace std;

int main()
{
    int i, n;
    cout << "enter number to check whether it is prime or not: ";
    cin >> i;
    for (n = 2; n < i; n++){
        if ((i % n) == 0)
        {
            cout << "it is not a prime number.";
            break;
        }
    }
    if (i == n){
        cout << "is a prime number."; 
        }
    return 0;
}

File: MLSA Event-101/Java/PrimeNumber.java
import java.util.*;

public class PrimeNumber {
    public static void main(String[] args) {
        Scanner s=new Scanner(System.in);
        int p;
        System.out.print("Enter number to check whether it is prime or not : ");
        p=s.nextInt();
        if(p==1){
            System.out.println("It is Prime !!");
        }
        else if(p==2){
            System.out.println("It is Prime !!");
        }
        for(int i=2;i<p;i++){
            if(p%i==0){
                System.out.println("Not Prime !!");
                break;
            }
            else{
                System.out.println("Prime !!");
                break;
            }
        }
        s.close();
    }
}
